polling_stations/apps/data_collection/management/commands/import_wiltshire.py

```diff
@@ -4,22 +4,31 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E06000054"
-    addresses_name = (
-        "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Wiltshire.tsv"
-    )
-    stations_name = (
-        "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Wiltshire.tsv"
-    )
-    elections = ["europarl.2019-05-23"]
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019wilt.tsv"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019wilt.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
 
```

Comment:
Should set `allow_station_point_from_postcode = False` unless there's  a specific reason not to.

Reply:
Yeah. I knew I'd missed this off, but you were too quick as I was coming back round to it :-/